data/en/mbs_0823.json

```diff
@@ -0,0 +1,375 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "survey_id": "009",
+    "theme": "default",
+    "data_version": "0.0.1",
+    "title": "Monthly Business Survey",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
+    "view_submitted_response": {
```

Comment:
Guess you'll have to add the eq_id and form_types to your schemas as well now

Reply:
Spoke with @nealedj about this and understood that we are not adding eq id and form types to schemas as this will be fed from upstream.